tests/integ/modin/frame/test_add_prefix.py

```diff
@@ -46,7 +46,7 @@ def test_df_add_prefix_multiindex(prefix, native_df_with_multiindex_columns):
     )
 
 
-@sql_count_checker(query_count=1)
+@sql_count_checker(query_count=1, join_count=1)
```

Comment:
do you know why it has one join?

Reply:
Join comes from `set_index` operation performed because we supply an index in the constructor

Reply:
This test should only update column names why it calls `set_index`?

Reply:
oh I see it is from `time_column_snowpark_pandas_df` construction.